### PR TITLE
Added language to benefits, reworked a couple of sections.

### PR DESCRIPTION
**Pull request by julianeon**

### CODE REVIEW

**Comment by stubailo** [comment]
Thanks!

File: index.html [hunk 1]
```diff
@@ -118,8 +118,8 @@ <h4 class="subtitle">What we do</h4>
       Meteor celebrates, supports, and thrives on the diversity of our employees, our
       products, and our community. We are committed to providing equal employment and advancement
       opportunities to qualified applicants regardless of their race, color, ancestry, gender, religion,
-      national origin, sexual orientation, age, citizenship, marital status, disability, Veteran or any
-      other legally protected status.
+      national origin, sexual orientation, age, citizenship, marital status, veteran status, disability or membership in any
+      legally protected category.
     </p>
 
     <h4 class="subtitle">Benefits & Perks</h4>
```
Comment (glasser) [inline index.html]: GitHub is highlighting this — is this required to be &amp; or is it OK?

Comment (stubailo) [inline index.html]: It renders just fine AFAIK

Comment (benjamn) [inline index.html]: Probably should be `&amp;` if we're being pedantic 😛

Comment (stubailo) [inline index.html]: Be happy to accept a PR from either of you :]

**Pull request merged**